test/interval_tests/numeric.jl

```diff
@@ -434,3 +434,30 @@ end
     @test nthroot(Interval{BigFloat}(-81, -16), -4) == ∅
     @test nthroot(Interval{BigFloat}(-81, -16), 1) == Interval{BigFloat}(-81, -16)
 end
+
+# approximation used for testing (not to rely on ≈ for intervals)
+# ⪆(x, y) = (x ≈ y) && (x ⊇ y)
+⪆(x::Interval, y::Interval) = x.lo ≈ y.lo && x.hi ≈ y.hi && x ⊇ y
+
+@testset "`mod`" begin
```

Comment:
Could you also add tests where `y` (the second argument) is an interval?

Reply:
It's not yet implemented, so I've added a todo and test that it throws `TypeError`.